### DISCUSSION TITLE
Integrating a new thermal model

### DISCUSSION THREAD

**Discussion by balaji3112** [pybamm-team/liionpack#233] [Q&A]
I'm planning to integrate a Thermal model with a Phase Change Material based cooling system to the li-ion pack model, It would be helpful if there is some reference out there for integrating a custom model inside the liionpack framework.

**Comment by TomTranter** [comment]
Make a new sim_func in your script and add model in there. See example: https://liionpack.readthedocs.io/en/latest/examples/06%20Changing%20a%20model/
